src/index.js

```diff
@@ -0,0 +1,367 @@
+            <li className="game-list-toggle">
+              <button
+                onClick={(e) => {
+                  const list = e.currentTarget.closest('ol');
+
+                  if (list.classList.contains('open')) {
+                    list.classList.remove('open');
+                  } else {
+                    list.classList.add('open');
+                  }
+                }}
+              >
+                List of Moves
+              </button>
+            </li>
+            {moves}
+          </ol>
+        </div>
+      </div>
+    );
+  }
+}
+
+class Board extends React.Component {
+  renderSquare(i, className) {
+    return (
+      <Square
+        className={className || 'square'}
+        value={this.props.squares[i]}
+        onClick={() => this.props.onClick(i)}
+        key={i}
+      />
+    )
+  }
+
+  render() {
+    let squares = [];
+    let rows = [];
+
+    for (let i = 0; i < 9; i++){
+      let className = 'square';
+
+      const winnerLine = this.props.winnerLine;
+      if (winnerLine && winnerLine.includes(i)) {
+        className = 'square square-win';
+      }
+
+      squares.push(this.renderSquare(i, className));
+    }
+
+    for (let i = 0; i < 3; i++){
+      rows.push(squares.slice(i*3, i*3+3))
+    }
+
+    return (
+      <div className="board-container">
+        <div className="board-row">
+          {rows[0]}
+        </div>
+        <div className="board-row">
+          {rows[1]}
+        </div>
+        <div className="board-row">
+          {rows[2]}
+        </div>
+      </div>
+    );
+  }
+}
+
+function GameModeBtm(props){
+  return (
+    <button
+      className={`game-mode ${props.active}`}
+      onClick={() => {
+          props.onClick();
+        }
+      }
+    >Click to play with {props.text}
+    </button>
+  )
+}
+
+function FirstStepRight(props){
+  return (
+    <div className={`first-step ${props.hide} ${props.lock}`}>
+      <h3 className="first-step__title">
+        Who will go first?
+      </h3>
+      <div className="first-step__btns">
+        <button
+          className="first-step__man-btn"
+          onClick={(e) => {
+              props.manFirst();
+              e.currentTarget.classList.add('active');
+              e.currentTarget.nextElementSibling.classList.remove('active');
+            }
+          }
+        >
+          <ManSVG />
+        </button>
+        <button
+          className="first-step__computer-btn"
+          onClick={(e) => {
+              props.computerFirst();
+              e.currentTarget.classList.add('active');
+              e.currentTarget.previousElementSibling.classList.remove('active');
+            }
+          }
+        >
+          <ComputerSVG />
+        </button>
+      </div>
+    </div>
+  )
+}
+
+function Square(props){
+  return (
+    <button className={props.className} onClick={props.onClick}>
+      {props.value}
+    </button>
+  );
+}
```

Comment:
you should also split this file in several, because it's a bad practice to write all code in 1 place

Reply:
Thanks, I fixed it